### PR TITLE
Theme improvements

### LINKED ISSUE #5: Add EGI branding to the theme
Add EGI branding to the theme.

- @dscardaci suggested to remove the list of sections/subsections at the bottom of pages

### LINKED ISSUE #249: Font size too small after theme update
# Short Description of the issue

Font size too small

### PR DESCRIPTION
**Pull request by thebe14**
# Addressing remaining open issues to the docs theme

- Updated the EGI logo in the top navbar, fixes #5 
- Increased the theme font some more, fixes #249 
- Fixed the **Copy** button on the code/shell examples. It no longer has a double shadow, is now more opaque so that it works well when overlaps text.

### CODE REVIEW

File: assets/scss/_variables_project.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@ $primary: rgb(80, 135, 180) !default;
 
 // Fonts
 
-$font-size-base: 0.8rem !default;
```
Comment (gwarf) [inline assets/scss/_variables_project.scss:13]: 🍾 :)

Comment (thebe14) [reply assets/scss/_variables_project.scss:13]: Yeah, but on my laptop with very high resolution on a smallish screen, I have to use scaling. Both on my laptop screen (250%) and on the external monitor connected to it (125%). And this docs site is unusable, unless zoom is at 70% or less. Especially the left sidebar (created issue to make it resizable at least).

**Comment by gwarf** [comment]
The white logo is certainly more readable, but it's not an official one, and not aligned to https://confluence.egi.eu/display/EGIG/Use+of+EGI%2C+EGI+Foundation+and+EGI+Federation.

But I guess with the issue with the background in blue, this would require a general update of the look and feel, so probably something that can be postponed to later. (maybe also once we will have adopted the next/future bootstrap 5 based docsytheme, cf https://github.com/google/docsy/discussions/1359)

So fine with me to merge this version, and then review the look and feel according to the guidelines we may/will get.

**Review by gwarf — APPROVED** [review]
LGTM

**Comment by thebe14** [comment]
Exactly. Let's also close the linked issues, we can revisit them later if needed.

**Comment by thebe14** [comment]
I am done

**Pull request merged**